test/ReactWrapper-spec.js

```diff
@@ -465,6 +465,30 @@ describeWithDOM('mount', () => {
         ));
         expect(foundNotSpan).to.have.length(0);
       });
+
+      it('should return props object when props() is called', () => {
+        const SFC = function SFC({ selector }) {
+          return (
+            <div data-foo={selector}>Test SFC</div>
+          );
+        };
+
+        const selector = 'blah';
+        const wrapper = mount(<SFC data-foo={selector} />);
```

Comment:
this is moot, as it doesn't really test the component itself, but you are implementing the SFC incorrectly. You are passing it `{data-foo: selector}` when the deconstructor in the props is looking for something more like `{selector: ...}`. Might be worth just having this be correct to prevent any sort of misunderstanding